### DISCUSSION TITLE
KICS Binaries Deprecation

### DISCUSSION THREAD

**Discussion by cx-nuno-oliveira** [Checkmarx/kics#4744] [Announcements]
Please be notified that KICS is **deprecating** the availability of binaries in the GitHub releases assets as of 1.5.0.
We intend to stop publishing the binaries along with **KICS 1.5.2 (scheduled for Mid of February)**.
It is advised to update all systems (pipelines, integrations, etc.) to use [KICS Docker Images](https://hub.docker.com/r/checkmarx/kics)

**Comment by JonZeolla** [comment]
@nunoocx can you please provide some additional context on this?  Interested in the reasoning behind this change.  Thanks!

**Reply by cx-joao-reigota** [reply]
Hi @JonZeolla,
KICS 1.5 introduced features that impact local environment requirements.
To make things easier for users, docker images provide all necessary requirements.

Let us know if you need any help/advise on moving your KICS usage to docker.
We will be glad to support you!

**Comment by createchange** [comment]
Sorry to resurrect a dead post, but is there any possibility of bringing back the binary releases?

If I want to enable my developers to shift left, I am left either building the binary and distributing it or, alternatively, using the Docker approach. Neither is a great solution, but Docker is especially problematic as we would have to be licensed for each and every developer.

I assume that the change that occurred around this time was the store of scan assets? If so, I have already concocted up a simple bash script that clones the assets repo down and ensure it is updated prior to scans. Building / distributing the binary is less straight-forward.
